reference/roadmap.md

```diff
@@ -12,5 +12,6 @@ Anbox Cloud follows a defined release cycle with frequent minor and patch releas
 
 | Release | Target Date | What to expect| 
 |---------|-------------|---------------|
-| 1.21.2 | April 18 2024 | * Android security updates for April 2024<br/>* Bug fixes |
 | 1.22.0 | May 15 2024 | * Support for custom Android system images<br/> * Support for 64 bit only ARM/x86 systems<br/>* Stability improvements for AAOS images and VHAL<br/>* VirGL enabled by default for NVIDIA GPUs<br/>* Update to NVIDIA 550 driver series |
```

Comment:
Unrelated to the changes, but shouldn't this also have the May security updates?

Reply:
Thanks for catching this, I have included it